helpers/env_vars.go

```diff
@@ -53,27 +60,88 @@ var (
 	TICSecretEnvVar = "TIC_SECRET"
 )
 
-// EnvVars holds all the environment variable values that a non-test server should have.
+// EnvVars provides a convenient method to access environment variables
 type EnvVars struct {
-	ClientID        string
-	ClientSecret    string
-	Hostname        string
-	LoginURL        string
-	UAAURL          string
-	APIURL          string
-	LogURL          string
-	PProfEnabled    string
-	BuildInfo       string
-	NewRelicLicense string
-	SecureCookies   string
-	LocalCF         string
-	SessionBackend  string
-	SessionKey      string
-	BasePath        string
-	SMTPHost        string
-	SMTPPort        string
-	SMTPUser        string
-	SMTPPass        string
-	SMTPFrom        string
-	TICSecret       string
+	path []EnvLookup
+}
+
+// Get returns value for key if present, else returns defaultVal if not found
+func (el *EnvVars) Get(key, defaultVal string) string {
+	rv, found := el.load(key)
+	if !found {
+		return defaultVal
+	}
+	return rv
+}
+
+// MustGet will panic if value is not set, otherwise it returns the value.
+func (el *EnvVars) MustGet(key string) string {
+	rv, found := el.load(key)
+	if !found {
+		panic(&ErrMissingEnvVar{EnvVar: key})
```

Comment:
@aeijdenberg you could remove the `EnvVar` stutter: 

```go
// ErrMissingEnvVar is panicked if a MustGet fails.
type ErrMissingEnvVar struct {
	// Name is the name of the env var that was not found
	Name string
}
```

Minor, but I used `name` not `key` - I thought the former was the correct terminology, but yeah it's minor.

Reply:
Fixed as suggested.